snuba/datasets/schemas/tables.py

```diff
@@ -222,7 +222,7 @@ def __init__(
         mandatory_conditions: Optional[Sequence[Condition]] = None,
         prewhere_candidates: Optional[Sequence[str]] = None,
         order_by: str,
-        partition_by: str,
+        partition_by: Optional[str],
```

Comment:
Where did this change come from?

Reply:
I just noticed the type error in vscode inline and since it was a trivial fix (the parent class is Optional and there is no reason to behave differently) I just applied it here.